src/app/pages/cinemahall/cinemahall.module.ts

```diff
@@ -0,0 +1,18 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { CinemahallComponent } from './cinemahall.component';
+
+
+
+@NgModule({
+  declarations: [
+    CinemahallComponent
+  ],
+  exports: [
+    CinemahallComponent
+  ],
+  imports: [
+    CommonModule
```

Comment:
I think it is an unused module.

Reply:
I will use it in a next MR

Reply:
You can forget to delete it in case solution design for the next MR is changed.
Now it is easy to track such modules but it will be harder as complexity of modules is growing. Avoid importing unused modules in future to prevent bad habits.

Reply:
YAGNI :)
Deleted